### DISCUSSION TITLE
Project and Community Structure

### DISCUSSION THREAD

**Discussion by afflom** [emporous/community#2] [General]
Hi Everyone,

I'd like to get this project and community kicked off by talking about structure. From what I can tell, I think that UOR is made up of the following core components:

1. Universal Object Reference Model (The UOR object network and attribute routing)
2. Universal Runtime (UOR Artifact embedded runtimes)
3. Provider integrations (OCI/IPFS/S3... )

Lets discuss if these core components are accurate or if we need to add or modify this list. Similarly, I think that UOR Special Interest Groups should govern each of these subjects and maybe even be subdivided by sub-components.

**Comment by usrbinkat** [comment]
Founding Tasks by category (as I see them)

Project:
- [x] Establish Project level project board
- [ ] Create template sig repo
- [x] Create .github organizational repo
- [ ] Create Community repo and traditional contents
  - establish community guidelines
  - draft the governance model
- [x] Create GH Pages repo for future docs
- [ ] Create the Framework repo from the template repo.
  - This likely should be a high-level introductory and documentation hub until a docs pages site takes shape.
- [ ] Define the scope of early work by sig. Fewer and smaller is better at this early stage.

Paper:
- [x] Agree upon location for the paper during the editing process
- [x] Define terms in a master Glossary
- [x] Review proposed index structure

**Reply by afflom** [reply]
I think that this is a great start. I'll create the project and throw these items on it for people to pickup

**Reply by afflom** [reply]
Ok, I've created the project and a few issues from the list.

**Comment by afflom** [comment]
I'm editing my comment here. 

Let's knock out the rest of the list!

**Comment by usrbinkat** [comment]
blocked by https://github.com/uor-framework/.github/pull/8
